Movapp (iOS).entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array/>
```

Comment:
Pls, potřeboval bych v Signing & Capabilities přidat nějakou AppGroup (abych mohl sdílet data v "UserDefault" napříč appkou a widgetem). (nejde to bez certifikátu).

Reply:
@zoul můžeš prosím zařídit? je to spojené s tímto #19

Reply:
Zkusil jsem, dejte pak (po rebase) vědět, jestli je to OK. Kdyby ne, reklamujte.

Reply:
(Pokud máte nějaké certifikáty nainstalované u sebe, aktualizujte si je, App Group vyžadovala přegenerování.)